server/stream.go

```diff
@@ -3218,9 +3218,8 @@ func (mset *stream) internalLoop() {
 					pm.o.didNotDeliver(pm.seq)
 				}
 				pm.returnToPool()
+				c.flushClients(0)
```

Comment:
Does this have an impact on perf for simple / normal setups?

Reply:
I have undone this and the stall client changes. I would rather introduce these kind of changes in a later release.